### PR TITLE
Added an output section to the help file

### PR DESCRIPTION
**Pull request by Jillderon**

### CODE REVIEW

**Comment by JorisGoosen** [comment]
@Kucharssim can you check whether the additions to the help make sense?

**Comment by boutinb** [comment]
@Kucharssim Should we add this to 0.15?

**Comment by Kucharssim** [comment]
When did this happen, I have no recollection of this PR 😅  I have a look

File: inst/help/networkanalysis.md [hunk 1]
```diff
@@ -28,20 +28,17 @@ This analysis allows you to estimate not one type of network, but multiple. Supp
...
 
 #### Plots
 - Network plot: a plot of the estimated network.
-- Centrality plot: a plot of centrality measures of the estimated network.
+- Centrality plot: a plot of the centrality measures of the estimated network.
+- Clustering plot: a plot of the clustering measures of the estimated network. 
 
 #### Tables
 - Centrality table: a table containing the values of the centrality measures.
+- Clustering table: a table ocntaining the values of the clustering measures.
```
Comment (Kucharssim) [inline inst/help/networkanalysis.md:40]:
```suggestion
- Clustering table: a table containing the values of the clustering measures. 
```

**Pull request merged**